tests/MonologCreator/Factory/HandlerTest.php

```diff
@@ -1,72 +1,57 @@
 <?php
+
 namespace MonologCreator\Factory;
 
-use \Monolog;
+use Monolog;
 
 /**
  * Class HandlerTest
  *
  * @package MonologCreator\Factory
  */
-class HandlerTest extends \PHPUnit_Framework_TestCase
+class HandlerTest extends \PHPUnit\Framework\TestCase
 {
     /**
      * @var \PHPUnit_Framework_MockObject_MockObject
      */
-    private $_mockFormatterFactory = null;
+    private $mockFormatterFactory = null;
 
     /**
      * @var \PHPUnit_Framework_MockObject_MockObject
      */
-    private $_mockFormatter = null;
+    private $mockFormatter = null;
 
     /**
      * @var \PHPUnit_Framework_MockObject_MockObject
      */
-    private $_mockUdpSocket = null;
+    private $mockUdpSocket = null;
 
     /**
      * @var \PHPUnit_Framework_MockObject_MockObject
      */
-    private $_mockPredisClient = null;
+    private $mockPredisClient = null;
 
     public function setUp()
     {
         parent::setUp();
 
-        $this->_mockFormatterFactory = $this->getMock(
-            '\MonologCreator\Factory\Formatter',
-            array(
-                'create',
-            ),
-            array(),
-            '',
-            false
-        );
-        $this->_mockFormatter = $this->getMock(
-            '\Monolog\Formatter\FormatterInterface',
-            array(
-                'format',
-                'formatBatch',
-            ),
-            array(),
-            '',
-            false
-        );
-        $this->_mockUdpSocket = $this->getMock(
-            '\Monolog\Handler\SyslogUdp\UdpSocket',
-            array(),
-            array(),
-            '',
-            false
-        );
-        $this->_mockPredisClient = $this->getMock(
-            '\Predis\Client',
-            array(),
-            array(),
-            '',
-            false
-        );
+        $this->mockFormatterFactory = $this->getMockBuilder('\MonologCreator\Factory\Formatter')
+            ->disableOriginalConstructor()
+            ->setMethods(['create'])
+            ->getMock();
+
+        $this->mockFormatter = $this->getMockBuilder('\Monolog\Formatter\FormatterInterface')
+            ->disableOriginalConstructor()
+            ->setMethods(['format', 'formatBatch'])
+            ->getMock();
+
+        $this->mockUdpSocket = $this->getMockBuilder('\Monolog\Handler\SyslogUdp\UdpSocket')
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $this->mockPredisClient = $this->getMockBuilder('\Predis\Client')
+        ->disableOriginalConstructor()
+        ->getMock();
```

Comment:
Indentation is wrong for these two lines.

Reply:
Also it feels odd to add a mock but not any methods it should have.

Reply:
fixed.